### ISSUE TITLE
Velp menua ei ole lokalisoitu

### ISSUE THREAD

**Issue by saviit** [TIM-JYU/TIM#3117]
Velp menu pitäisi lokalisoida tällä hetkellä tuetuille kielille (suomi ja ruotsi).
Uudemmissa commiteissa velp menun osakomponentteja on lokalisoitu jonkin verran, mutta velp menua ei kokonaisuutena ole käyty läpi ja lokalisoitu.

**Comment by dezhidki** [comment]
Täydellinen lokalisointi vaatisi varmaan ensin Velp menun muuntamista Angular-komponentiksi (osa korttia #496)

**Comment by saviit** [comment]
Joo, tämähän tulisi varmaan aika luonnollisesti osana tuota, lokalisoitavia merkkijonoja ei kuitenkaan niin paljoa ole.
